agimus_demo_01_lfc_alone/launch/bringup.launch.py

```diff
@@ -125,14 +132,28 @@ def prepare_launch_description():
     
     load_joint_state_broadcaster = ExecuteProcess(
         cmd=['ros2', 'control', 'load_controller', '--set-state', 'active',
-                'joint_state_broadcaster'],
+             'joint_state_broadcaster'],
         output='screen'
     )
-    
-    joint_impedance_example_controller = ExecuteProcess(
-        cmd=['ros2', 'control', 'load_controller', '--set-state', 'active',
-                'joint_impedance_example_controller'],
-        output='screen'
+
+    joint_state_estimator_yaml = str(
+        Path(get_package_share_directory("agimus_demo_01_lfc_alone")) /
+        "config" /
+        "joint_state_estimator.yaml"
+    )
```

Comment:
```suggestion
    joint_state_estimator_params = PathJoinSubstitution(
        [
            FindPackageShare("agimus_demo_01_lfc_alone"),
            "config",
            "joint_state_estimator.yaml",
        ]
    )
```
This is a proper way of handling paths in ROS 2. This also requires this include
```python
from launch.substitutions import PathJoinSubstitution
from launch_ros.substitutions import FindPackageShare
```